platform/search/bundle.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "constants": [
+            {
+                "key": "GENERIC_SEARCH_ROOTS",
```

Comment:
Ah, I see-- definitely a misunderstanding-- didn't know that "ROOT" was a domain object.

From my reading of the docs, `roots[]` is the defacto way that you would access all members of an extension category.  Moreso, `roots[]` is based on a key framework concept-- extension categories, and I feel like we should focus on repeating common patterns instead of having specialized concepts like "ROOT".  In this case, I'm also concerned that "ROOT" is an implementation detail-- not something that we should be considering part of the framework's API.